Packs/Zoom/Integrations/Zoom/Zoom.py

```diff
@@ -1,148 +1,621 @@
+            'Metadata': {'Count': raw_data.get('page_count'),
+                         'Number': raw_data.get('page_number'),
+                         'Size': raw_data.get('page_size'),
+                         'Total': raw_data.get('total_records')}
+        },
+        raw_response=raw_data
+    )
+
+
+def zoom_create_user_command(client: Client, user_type: str, email: str, first_name: str, last_name: str) -> CommandResults:
+    user_type_num = USER_TYPE_MAPPING.get(user_type)
+    raw_data = client.zoom_create_user(user_type_num, email, first_name, last_name)
+    return CommandResults(
+        outputs_prefix='Zoom.User',
+        readable_output=f"User created successfully with ID: {raw_data.get('id')}",
+        outputs=raw_data,
+        raw_response=raw_data
+    )
+
+
+def zoom_delete_user_command(client: Client, user_id: str, action: str) -> CommandResults:
+    client.zoom_delete_user(user_id, action)
+    return CommandResults(
+        outputs_prefix='Zoom',
+        readable_output=f'User {user_id} was deleted successfully',
+    )
+
+
+def zoom_create_meeting_command(
+        client: Client,
+        user_id: str,
+        topic: str,
+        host_video: bool | str = True,
+        jbh_time: int | str | None = None,
+        start_time: str = None,
+        timezone: str = None,
+        type: str = "instant",
+        auto_record_meeting: str = "none",
+        encryption_type: str = "enhanced_encryption",
+        join_before_host: bool | str = False,
+        meeting_authentication: bool | str = False,
+        waiting_room: bool | str = False,
+        end_date_time: str | None = None,
+        end_times: int | str = 1,
+        monthly_day: int | str = 1,
+        monthly_week: int | str | None = None,
+        monthly_week_day: int | str | None = None,
+        repeat_interval: int | str | None = None,
+        recurrence_type: int | str | None = None,
+        weekly_days: int | str = 1) -> CommandResults:
+
+    # converting
+    host_video = argToBoolean(host_video)
+    join_before_host = argToBoolean(join_before_host)
+    meeting_authentication = argToBoolean(meeting_authentication)
+    waiting_room = argToBoolean(waiting_room)
+    end_times = arg_to_number(end_times)
+    monthly_day = arg_to_number(monthly_day)
+    monthly_week = arg_to_number(monthly_week)
+    monthly_week_day = arg_to_number(monthly_week_day)
+    repeat_interval = arg_to_number(repeat_interval)
+    weekly_days = arg_to_number(weekly_days)
+
+    num_type = 1  # "instant"
+    if type == "scheduled":
+        num_type = 2
+    elif type == "recurring meeting with fixed time":
+        num_type = 8
+
+    # argument checking
+    # for arguments that have a default value, i use demisto.args to trigger an exception if user entered a value.
+    args = demisto.args()
+    if type == "instant" and (timezone or start_time):
+        raise DemistoException("Too money arguments. start_time and timezone are for scheduled meetings only.")
```

Comment:
Lets take the error to a const.

```suggestion
        raise DemistoException("Too many arguments. Please use start_time and timezone for scheduled meetings only.")
```